api/krusty/fnplugin_test.go

```diff
@@ -630,3 +632,194 @@ metadata:
   name: env
 `)
 }
+
+func TestFnContainerMounts(t *testing.T) {
+	skipIfNoDocker(t)
+
+	th := kusttest_test.MakeHarness(t)
+	o := th.MakeOptionsPluginsEnabled()
+	fSys := filesys.MakeFsOnDisk()
+	b := MakeKustomizer(&o)
+	tmpDir, err := filesys.NewTmpConfirmedDir()
+	assert.NoError(t, err)
+
+	path, err := os.Getwd()
+	assert.NoError(t, err)
+
+	src := filepath.Join(path, "testdata", "charts") + string(filepath.Separator)
+	dst := filepath.Join(tmpDir.String(), "testdata", "charts") + string(filepath.Separator)
+	assert.NoError(t, os.MkdirAll(dst, 0777))
+
+	if runtime.GOOS == "darwin" {
+		cmd := exec.Command("cp", "-r", src, dst)
+		assert.NoError(t, cmd.Run())
+	} else if runtime.GOOS == "linux" {
+		d := dst + "."
+		cmd := exec.Command("cp", "-r", src, d)
+		assert.NoError(t, cmd.Run())
+	} else {
```

Comment:
This test won't get run on Windows. But since the krusty tests don't run in the Windows CI job, and neither of us have a Windows machine, I don't have any method of actually verifying that whatever I write here would work on windows. I'm thinking we can reenable this once we're able to run Windows tests somewhere.